scripts/execute_ebpf_cicd_tests.ps1

```diff
@@ -20,19 +20,8 @@ $AdminTestVMCredential = Get-StoredCredential -Target $AdminTarget -ErrorAction
 $StandardUserTestVMCredential = Get-StoredCredential -Target $StandardUserTarget -ErrorAction Stop
 
 # Load other utility modules.
-Import-Module .\common.psm1 -Force -ArgumentList ($LogFileName) -WarningAction SilentlyContinue
-Import-Module .\vm_run_tests.psm1  `
-    -Force `
-    -ArgumentList (
-        $AdminTestVMCredential.UserName,
-        $AdminTestVMCredential.Password,
-        $StandardUserTestVMCredential.UserName,
-        $StandardUserTestVMCredential.Password,
-        $WorkingDirectory,
-        $LogFileName,
-        $TestHangTimeout,
-        $UserModeDumpFolder) `
-    -WarningAction SilentlyContinue
+Import-Module $PSScriptRoot\common.psm1 -Force -ArgumentList ($LogFileName) -WarningAction SilentlyContinue
+Import-Module $PSScriptRoot\vm_run_tests.psm1  -Force -ArgumentList ($AdminTestVMCredential.UserName, $AdminTestVMCredential.Password, $StandardUserTestVMCredential.UserName, $StandardUserTestVMCredential.Password, $WorkingDirectory, $LogFileName) -WarningAction SilentlyContinue
```

Comment:
this looks like a bad merge - you're removing the parameters that Dhiren recently added for dump colleciton on test timeout.
Also, the old formatting was better.

Reply:
this is the automated source formatting, so is that a bad thing to have? should we disable it?

Reply:
I'd recommend a test run that actually generates dumps (Set TestHangTimeout to 30 and run the cicd stress test) to make sure the dumps are generated and collected as expected.   

This is subjective, but re. formatting, the automated formatting fixes deviations from the conventions that we follow.  It does not stop us from making the contents more 'readable' :-)

Reply:
oook :)

Reply:
You didn't address the main par tof the comment, iee. the removal of dump-related parameters.

Reply:
I think you're looking at the outdated diff in the comment... please look at the diff in the PR:

<img width="1352" alt="image" src="https://github.com/microsoft/ebpf-for-windows/assets/22826130/8d428b1e-c0b1-4d3f-b10b-b8b52565d952">